vue/components/ui/molecules/input-color/input-color.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+    <div class="input-color-container" v-bind:class="classes">

```

Comment:
Can we add some kind of visual feedback for when there's an error? (invalid color etc) Showing an empty color box, for instance.